common/src/main/java/com/revenuecat/purchases/common/subscriberattributes/SpecialSubscriberAttributes.kt

```diff
@@ -1,40 +1,40 @@
-package com.revenuecat.purchases.subscriberattributes
+package com.revenuecat.purchases.common.subscriberattributes
 
 /**
  * Special Attributes
  */
-internal const val SPECIAL_KEY_EMAIL = "\$email"
-internal const val SPECIAL_KEY_DISPLAY_NAME = "\$displayName"
-internal const val SPECIAL_KEY_PHONE_NUMBER = "\$phoneNumber"
-internal const val SPECIAL_KEY_FCM_TOKENS = "\$fcmTokens"
+const val SPECIAL_KEY_EMAIL = "\$email"
+const val SPECIAL_KEY_DISPLAY_NAME = "\$displayName"
+const val SPECIAL_KEY_PHONE_NUMBER = "\$phoneNumber"
+const val SPECIAL_KEY_FCM_TOKENS = "\$fcmTokens"
 
 /**
  * Device Identifiers
  */
-internal const val SPECIAL_KEY_IDFA = "\$idfa"
-internal const val SPECIAL_KEY_IDFV = "\$idfv"
-internal const val SPECIAL_KEY_IP = "\$ip"
-internal const val SPECIAL_KEY_GPS_AD_ID = "\$gpsAdId"
-internal const val SPECIAL_KEY_ANDROID_ID = "\$androidId"
+const val SPECIAL_KEY_IDFA = "\$idfa"
+const val SPECIAL_KEY_IDFV = "\$idfv"
+const val SPECIAL_KEY_IP = "\$ip"
+const val SPECIAL_KEY_GPS_AD_ID = "\$gpsAdId"
+const val SPECIAL_KEY_ANDROID_ID = "\$androidId"
```

Comment:
these are not internal anymore, bu they are actually in the `common` module, so it should be fine

Reply:
not needed for this PR, but perhaps we could take the opportunity to rename them `reserved` instead of `special`, like we did for iOS?